### DISCUSSION TITLE
Local repositories cannot be created

### DISCUSSION THREAD

**Discussion by distributev** [semaphoreui/semaphore#1537] [Q&A]
![image](https://github.com/ansible-semaphore/semaphore/assets/19224635/bbb4a21a-099d-4098-aeb3-565d417ac144)

**Comment by distributev** [comment]
Hello, do you know if I'm doing something wrong or this indeed doesn't work? If so, are there workarounds to this?

**Comment by NikitaIVll** [comment]
you can create a new key and make it None

**Comment by fiftin** [comment]
Hi @distributev 
You need manually create None key. I will add None key by default in next release.
